python/openmldb/sdk/__init__.py

```diff
@@ -11,3 +11,5 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+
+from . import sdk
```

Comment:
We may change `from sdk import *` later and update the usage in `dbapi.py` or others.

Reply:
updated.